README.md

```diff
@@ -3,21 +3,28 @@ go-ipld-eth-import
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
 [![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
-[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
 
 > go-ipld-eth-import is the set of tools that will help us to bring the Ethereum magic to the IPLD merkle forest.
 
+## Project Status
+
+Please note that this project is **not actively maintained**.
+
+For the latest IPLD for Ethereum see:
+
+* Specification: https://ipld.io/specs/codecs/dag-eth/
+* Codec: https://github.com/vulcanize/go-codec-dageth
+
+https://github.com/vulcanize/eth-block-extractor/ is a similar project to this but is also not actively maintained, these two projects may be merged into one, or one of them retired when / if work is resumed.
+
+If you are interested in contributing to further development of this project, please open an issue for discussion, we would love to hear from you!
```

Comment:
It's not possible to open issues on archived repos. Maybe just point people to https://discuss.ipfs.io/?